src/Controller/SingleRecipeController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Recipe;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\Routing\Annotation\Route;
+
+class SingleRecipeController extends AbstractController
+{
+    /**
+     * @Route("/single/recipe/{id}", name="single_recipe")
```

Comment:
pavyzdis routo is dokumentacijos 
```
    /**
    * @Route("/blog/{page}", name="blog_list", requirements={"page"="\d+"})
     */
    public function list(int $page)
    {
        // ...
    }
```

tada `public function index(int $id)` ir nebereikia `intval`

Reply:
o dar gal gali i slacka prisijungt? man ten errora meta. Suvedus patha nueiti galima, bet tos korteles nesukuria kur patha bandau padaryt

Reply:
beje taisiklingas route name turetu but: `/recipes/{id}`